### DISCUSSION TITLE
My GitHub pages is getting marked as Dangerous by Chrome

### DISCUSSION THREAD

**Discussion by rohit0235** [community/community#123231] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My GitHub pages is getting marked as Dangerous by Chrome

**Comment by krishealty** [comment]
well your github.io domain landed on **Google's Safe Browsing blocklist**. I've seen a lot of cases around this forum where that happened to people after they put a clone of some popular website (or at least its login page) up, which then counts as phishing.

I have attached a screenshot of what i suppose a better answer of the problem, and a link where the same question is asked.

[https://support.google.com/webmasters/thread/171013030/my-github-hosted-pages-are-showing-deceptive-site-ahead?hl=en](url)

![Screenshot_2024-05-11-10-19-49-36_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/community/community/assets/93068820/4d677b3d-468e-47c4-9834-0a7d0cc2f1ac)


you can refer to the below guidelines of Google:

[https://developers.google.com/search/docs/monitor-debug/security/social-engineering?hl=en](url)

**Reply by LiteBrite82** [reply]
Hi @rohit0235, 

@krishealty is correct; though I don't know what happened to that linked discussion; it was a good one. Depending on the exact error you receive, it could also be something like DNS verification or HTTPS issues. 

Can you provide screenshots or links?

**Reply by rohit0235** [reply]
thanks for your reply . now my issue is resolved

**Reply by krishealty** [reply]
@LiteBrite82 thanks for pointing it out, i have updated my answer for the future developers, if anyone has the same problem.
